client/components/App.jsx

```diff
@@ -21,18 +22,32 @@ class App extends Component {
     this.paginate = this.paginate.bind(this);
     this.incrementPage = this.incrementPage.bind(this);
     this.decrementPage = this.decrementPage.bind(this);
+    this.onSortReviews = this.onSortReviews.bind(this);
   }
 
   componentDidMount() {
     this.getReviews();
   }
 
+  onSortReviews(sortQuery) {
+    const { reviews } = this.state;
+    let sorted;
+    if (sortQuery === 'Newest') {
+      sorted = reviews.sort((a, b) => new Date(b.date) - new Date(a.date));
+    } else {
+      sorted = reviews.sort((a, b) => b.recommended - a.recommended);
+      console.log(sorted);
+    }
+
+    this.setState({ reviews: sorted, currentPage: 1 });
+  }
+
```

Comment:
Unrelated to this block of code... I started up your api and noticed the left button was clickable a few times and loaded a page with no reviews. Every thing looks awesome!!!